src/modules/SystemSettings.rb

```diff
@@ -287,11 +289,17 @@ def device_supports_scheduler(device, scheduler)
     # @see Read
     def read_scheduler
       # Read bootloader settings in normal mode
-      Bootloader.Read if Mode.normal
+      if Mode.normal
+        bootloader_read = Bootloader.Read
+
+        return false unless bootloader_read
```

Comment:
Kind of similar to the previous one, I would clarify here that `Bootloader.Read` is expected to already had reported the problem to the user... although maybe is not needed, since is kind of the traditional YaST approach. Whatever, feel free to ignore this.